packages/core/addon/serializers/dashboard.js

```diff
@@ -84,9 +86,10 @@ export default AssetSerializer.extend({
     dashboard.data.attributes.filters = dashboard.data.attributes.filters.map((filter) => {
       const source = filterSources[buildKey(filter)];
       return {
+        type: filter.type,
```

Comment:
Can we run the elide NetflixTitles demo and try saving a dashboard with a time filter. I thought we were stuck with the existing fields because elide wouldn't save them
https://github.com/yavin-dev/framework/blob/aebf77433f79c9136579fa0f951f303b3756107c/packages/webservice/models/src/main/kotlin/com/yahoo/navi/ws/models/beans/fragments/request/DashboardFilter.kt#L7-L12

Reply:
![image](https://user-images.githubusercontent.com/13946669/151431344-ab3058e8-b48b-41bb-b266-6ba5c7c759a6.png)